sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -160,7 +160,7 @@ must be passed to the clang driver:
 
 `-fsycl`
 
-With this option specified, the driver will invoke the host SYCL compiler and a
+With this option specified, the driver will invoke the host compiler and a
 number of device compilers for targets specified in the `-fsycl-targets`
```

Comment:
Instead of removing it for the host you could add this :-)
```suggestion
number of SYCL device compilers for targets specified in the `-fsycl-targets`
```

Reply:
*SYCL* device compiler is a good addition (referring to SYCL device concept), for host it's not so obvious, imho host is just host everywhere :)